### ISSUE TITLE
I trained the ntu120 data joint mode on X-sub according to README.md, but the accuracy is only in the 44%, is this because there is something missing in the code?

### ISSUE THREAD

**Issue by blue-q** [damNull/LAGCN#8]

**Comment by damNull** [comment]
The code repo is complete and reproducible. The inconsistent performance may due to data processing error. You can refer to issue #6 and #7.

**Comment by blue-q** [comment]
Hi @damNull, thanks for your reply. How much "loop_times"  should be set in the training process?

**Comment by damNull** [comment]
The experiments about `loop_times` are in Table 5 of Arxiv article.

**Comment by blue-q** [comment]
@damNull  yeah, I now set `loop_times `to 0,1,2 respectively for training, both on ntu60 and 120 with only 40 + accuracy. I only modified the value of `loop_times `in the model, and I also processed the data according to readme.md. Could you please help me analyze the reason?
